raspibolt-pulse-switch.sh

```diff
@@ -10,13 +10,14 @@ set -u
 
 # set datadir
 bitcoin_dir="/data/bitcoin"
-lnd_dir="/data/lnd"
+# Chose between LND and CLN
+ln_implemenation="CLN"
 
 # set to mount point of secondary storage. This is used to calculate secondary USB usage %
 ext_storage2nd="/mnt/ext"
 
 # set to network device name (usually "eth0" for ethernet, and "wlan0" for wifi)
-network_name="wlan0"
+network_name="eth0"
```

Comment:
```suggestion
network_name="eth0"
```

I suggest `wlan0 `is the default value as Wifi is the suggested method in the main guide (although IMO I think we should encourage people to use ethernet personnaly!)

Reply:
In the telegram poll, ethernet was the favorite, but I can set it to wlan anyways, wdyt?

Reply:
> In the telegram poll, ethernet was the favorite, but I can set it to wlan anyways, wdyt?

Yes, there is clear majority of users that use ethernet rather than WiFi: https://t.me/raspibolt/13297

I'd vote for having `eth0 `as default now 👍